functions/src/util.ts

```diff
@@ -51,3 +52,11 @@ export function getDocumentId(change: Change<DocumentSnapshot>): string {
   }
   return change.before.id
 }
+
+/**
+ * getSearchableFields
+ * @return {string[]} Fields
```

Comment:
Should the return type also be provided in the other functions? Maybe its for another PR though

Reply:
Oh yes I will create a PR to fix all the Jsdoc